src/session_impl.cpp

```diff
@@ -5548,6 +5554,28 @@ namespace aux {
 			, boost::ref(m_alerts), _1, cb), salt);
 	}
 
+	void session_impl::dht_get_peers(sha1_hash const& info_hash)
+	{
+		if (!m_dht) return;
```

Comment:
it's probably a good idea to have a clear response here. i.e. one call to dht_get_peers() results in one response alert, regardless whether it carries an error or a successful response.